lib/manager/bundler/locked-version.ts

```diff
@@ -0,0 +1,43 @@
+import { isVersion } from '../../versioning/ruby';
+
+const DEP_REGEX = new RegExp('(?<=\\().*(?=\\))');
+export function extractLockFileEntries(
+  lockFileContent: string
+): Map<string, string> {
```

Comment:
What's the reasoning behind choosing a `Map` type?

Reply:
Little more performant, than plain object